### ISSUE TITLE
Default Wallet Networks/RPCs

### ISSUE THREAD

**Issue by wottpal** [brave/brave-browser#20143]
I think your native browser wallet is a unique chance to make crypto easier for non-crypto-natives. But please don't clone MetaMask shortcomings. Nobody – except developers – need all those testnets. Instead you could **pre-configure** more relevant L1/L2s like Polygon or Fantom. You could also hide testnets under a collapsable category "Testnets".

<img width="362" alt="Screenshot 2021-12-16 at 20 46 19" src="https://user-images.githubusercontent.com/3930150/146438395-3968c38e-1df2-44e6-bd58-565ad84dc612.png">

I also have a lot more ideas about how your wallet could shine. For example:

* Why not show an icon for the currently selected network in the toolbar (next to the wallet icon) as well? If you want to make crypto a first-level citizen I want to know which chain I'm on without toggling the popover (which otherwise "just" feels like an extension).
* Integrate with https://tokenlists.org/ (and other L2-solutions) to auto-configure coins/tokens
* Integrate a bridge directly into the wallet (or better a bridge-aggregator to ensure low fees)
* Please show me a list of all my ERC-20 tokens when I click on the wallet icon – Showing only the network coin (i.e. ETH on mainnet, MATIC on polygon) makes absolutely no sense in our tokenized world. I don't want to maximize the window to be able to see that. When showing ERC-20 you can make **a LOT** things better than MetaMask: Sort by liquidity or let me manually sort tokens and fade out tokens with zero liquidity.
* https://github.com/brave/brave-browser/issues/19544
* Aggregate cross-chain-liquidity (show my overall balance)
* One of your team-members should have a look on zapper.fi or zerion.io. The best what could happen to crypto would be having those in a condensed way as a native browser wallet.

Should I create more issues? I really want Brave to get this right as I fell in love with the browser. ❤️

FYI @bbondy

**Comment by bbondy** [comment]
Yes! We're aligned on everything you said and working towards it all too.
